src/index.js

```diff
@@ -70,7 +70,7 @@ function AppWrapper() {
         <Web3Wrapper>
           <App>
             <Route path="/" exact component={Home} />
-            <Route path="/deposit" exact component={StartDeposit} />
+            {/* <Route path="/deposit" exact component={StartDeposit} />
```

Comment:
If we decide to remove the code instead of commenting, we should remove the code related to the `StartDeposit` component.